demo/EV3D4/README.md

```diff
@@ -0,0 +1,23 @@
+# EV3D4
+EV3D4 is designed to look like R2-D2 from Star Wars. There are two options for controlling EV3D4. The first is to use the IR remote to send commands to the IR sensor, run EV3D4RemoteControl to use this method. The second means of controlling EV3D4 is via a web browser, run EV3D4WebControl to use this method.
+
+**Building instructions**: https://www.lego.com/en-us/mindstorms/build-a-robot/ev3d4
+
+### EV3D4RemoteControl
+If you are using EV3D4RemoteControl the only other file you need from ```demo/EV3D4/``` is EV3D4.py. The other files are only needed by EV3D4WebControl.
+
+EV3D4RemoteControl creates a child class of ev3dev/helper.py's RemoteControlledTank.
```

Comment:
Paths here and below should be code-formatted as well; <kbd>Ctrl</kbd>+<kbd>F</kbd> for `/`